
<launch>
    <arg name="enable_imu_trigger" default="false"/>
    <node pkg="nodelet" type="nodelet" name="vision_nodelet" args="manager"
          output="screen"/>
    <node pkg="nodelet" type="nodelet" name="hk_camera"
          args="load hk_camera/HKCameraNodelet vision_nodelet"
          output="screen">
        <param name="camera_name" value="hk_camera"/>
        <param name="image_width" value="1440"/>
        <param name="image_height" value="1080"/>
        <param name="pixel_format" value="bgr8"/>
        <param name="camera_info_url" value=""/>
        <param name="camera_frame_id" value="camera_optical_frame"/>
        <param name="camera_sn" value="00J67375789"/>
        <param name="frame_rate" value="200"/>
        <param name="enable_imu_trigger" value="$(arg enable_imu_trigger)"/>
        <param name="gain_value" value="14"/>
        <param name="gain_auto" value="false"/>
    </node>
</launch>
